Sources/OutputTarget.swift

```diff
@@ -32,7 +32,7 @@ import Darwin.C
 
 private func getEnvValue(key: String) -> String? {
     let value = getenv(key)
-    return value != nil ? String.fromCString(value) : nil
+    return value != nil ? String(cString: value!) : nil
```

Comment:
This is the only line that makes me nervous since I'm not sure of the consequences of force unwrapping it (can't really understand the syntax).